pkg/api/helpers.go

```diff
@@ -577,7 +577,7 @@ func SysctlsFromPodAnnotation(annotation string) ([]Sysctl, error) {
 	sysctls := make([]Sysctl, len(kvs))
 	for i, kv := range kvs {
 		cs := strings.Split(kv, "=")
-		if len(cs) != 2 {
+		if len(cs) != 2 || len(cs[0]) == 0 {
```

Comment:
Does cs[1] can be empty ?
Actually when cs[0] is empty or invalid name format, it can be validated now but cs[1] can't. http://pastebin.test.redhat.com/417014